### PR TITLE
fix/CMIML-167-transl-and-layout-fixes

### PR DESCRIPTION
**Pull request by kdanilenka32**

### CODE REVIEW

**Review by nmialik — APPROVED** [review]

File: src/components/LeftBar/LeftBar.styles.ts [hunk 1]
```diff
@@ -1,17 +1,14 @@
...
 
 import theme from 'styles/theme';
 import { variables } from 'styles/variables';
 
-import { LEFT_BAR_WIDTH } from './LeftBar.const';
-
-export const StyledDrawer = styled(Drawer)`
-  .drawer-paper {
-    background-color: ${variables.palette.surface1};
-    width: ${LEFT_BAR_WIDTH};
```
Comment (IhorNesterenko99) [inline src/components/LeftBar/LeftBar.styles.ts:12]: Why did you remove variable? It helps to define width of the navigation panel in one place and if it changed, also changed left margin for the other content.

**Review by IhorNesterenko99 — APPROVED** [review]
